cli/admin/src/main/java/io/pravega/cli/admin/utils/ZKHelper.java

```diff
@@ -131,6 +135,32 @@ public String getMetaDataForReader(String readerPath) throws Exception {
         }
     }
 
+    /**
+     * Get the pending events detail for a request in a particular controller host.
+     * @param readerPath Path of reader.
+     * @throws Exception If unable to get the znode for readerpath.
+     * @return Event detail for a request.
+     */
+    public String getPendingEventsForRequest(String readerPath) throws Exception {
```

Comment:
what `exception` are we expecting here? can we be specific?

Reply:
This has a call to CuratorFrameWork   "zkClient.getData().forPath(readerPath);"  
and forPath throws Exception . Thus any Exception thrown will be handled.